src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs

```diff
@@ -83,13 +91,38 @@ private string GetArtifactRestAPIForRepo(string repoName)
 
         public async Task RunPipeline(string pipelineName, string reviewDetails, string originalStorageUrl)
         {
-            SetDevopsClientHeaders();
             //Create dictionary of all required parametes to run tools - generate-<language>-apireview pipeline in azure devops
             var reviewDetailsDict = new Dictionary<string, string> { { "Reviews", reviewDetails }, { "APIViewUrl", _hostUrl }, { "StorageContainerUrl", originalStorageUrl } };
-            var pipelineParams = new Dictionary<string, Dictionary<string, string>> { { "templateParameters", reviewDetailsDict } };
-            var stringContent = new StringContent(JsonSerializer.Serialize(pipelineParams), Encoding.UTF8, "application/json");
-            var response = await _devopsClient.PostAsync(_pipeline_run_rest, stringContent);
-            response.EnsureSuccessStatusCode();
+            var devOpsCreds = new VssBasicCredential("nobody", _configuration["Azure-Devops-PAT"]);
+            var devOpsConnection = new VssConnection(new Uri($"https://dev.azure.com/azure-sdk/"), devOpsCreds);
+            string projectName = _configuration["Azure-Devops-internal-project"] ?? "internal";
```

Comment:
We should use a definition id instead of a name as it will be more stable. Also just for clarity is this the project or pipeline name?

Reply:
project name. `internal`